README.md

```diff
@@ -381,54 +381,54 @@ Built-in functions. Ready to be bound to any key you like. :smile:
-| `builtin.help_tags`                 | Lists Available help tags and open help document.                                           |
-| `builtin.man_pages`                 | Lists Man entries.                                                                          |
-| `builtin.marks`                     | Lists Markers and their value.                                                              |
-| `builtin.colorscheme`               | Lists Colorscheme and switch to it on enter.                                                |
-| `builtin.quickfix`                  | Lists items from quickfix.                                                                  |
-| `builtin.loclist`                   | Lists items from current window's location list.                                            |
-| `builtin.vim_options`               | Lists vim options and on enter edit the options value.                                      |
-| `builtin.registers`                 | Lists vim registers and edit or paste selection.                                            |
-| `builtin.autocommands`              | Lists vim autocommands and go to their declaration.                                         |
-| `builtin.spell_suggest`             | Lists spelling suggestions for <cword>.                                                     |
-| `builtin.keymaps`                   | Lists normal-mode mappings.                                                                 |
-| `builtin.filetypes`                 | Lists all filetypes.                                                                        |
-| `builtin.highlights`                | Lists all highlights.                                                                       |
-| `builtin.current_buffer_fuzzy_find` | Searches in current buffer lines.                                                           |
-| `builtin.current_buffer_tags`       | Lists Tags in current buffer.                                                               |
+| Functions                           | Description                                                                                                      |
+|-------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| `builtin.buffers`                   | Lists open buffers in current neovim instance                                                                    |
+| `builtin.oldfiles`                  | Lists previously open files                                                                                      |
+| `builtin.commands`                  | Lists available plugin/user commands and runs them on <cr>                                                       |
+| `builtin.tags`                      | Lists tags in current directory with preview (ctags -R)                                                          |
```

Comment:
ctags need to manually generated, I think this is something important to mention in order to manage expectations. 

_Side note: God I always hated this. I'm sure whoever decided to neglect the generation of tags to the user, actually doesn't use tags 😒_

Reply:
`Lists tags in current directory with preview (ctags -R), so long as you manually generated them yourself already`

Do you think that wording is better? ^

Reply:
> `Lists tags in current directory with preview (ctags -R), so long as you have already manually generated them before`
> 
> 
> 
> Do you think that wording is better? ^

Perhaps:

Lists tags in current directory with tag location file preview (users are required to run `ctags -R` to generate tags or update when introducing new changes)

Reply:
Sounds good to me!